index.js

```diff
@@ -15,11 +15,15 @@
 const { AssetCompute } = require("./lib/assetcompute");
 const { AssetComputeEventEmitter } = require("./lib/eventemitter");
 const { AssetComputeClient } = require("./lib/client");
+const { AssetComputeClientRetry } = require("./lib/client-retry");
 const { getIntegrationConfiguration } = require('./lib/integrationConfiguration');
+const { TooManyRequestsError } = require('./lib/error');
 
 module.exports = {
     AssetCompute,
     AssetComputeEventEmitter,
     AssetComputeClient,
-    getIntegrationConfiguration
+    AssetComputeClientRetry,
+    getIntegrationConfiguration,
+    TooManyRequestsError
```

Comment:
Eventually, should make its way to the asset-compute-commons error file/lib.

Reply:
I was thinking that, but it is very specific to the asset compute client (and the lib doesn't have a dep on commons yet)